tests/unit/math/conftest.py

```diff
@@ -0,0 +1,17 @@
+import pytest
```

Comment:
I am not sure if it makes sense to add a conftest at this level. These fixtures can be added directly to the specific test

Reply:
true, it was done to share fixtures for the files in math, for now we can keep it in the same test.